tests/pluginMetricsDefault.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { register } from 'prom-client';
+import { Application } from 'express';
+import { IBasicAuth, IStorageManager, PluginOptions } from '@verdaccio/types';
+
+import MetricsPlugin, { DEFAULT_METRICS_PATH } from '../src';
+import { MetricsConfig } from '../src/types';
+
+// @ts-ignore
```

Comment:
Why does this need to be ignored?

Reply:
This was because the `compilerOptions.rootDir` was set to `src` in the `tsconfig.json` file and the import here was for a file under the `tests` directory. The error was: 
> TS6059: File 'tests/testUtils.ts' is not under 'rootDir'

It looks like just removing the `compilerOptions.rootDir` config resolves the issue without any negative side effects.